### ISSUE TITLE
just submitting bug fixes for your documentation

### ISSUE THREAD

**Issue by kinganu** [makinacorpus/django-safedelete#83]
on this page: http://django-safedelete.readthedocs.io/en/latest/managers.html , need this import, otherwise you get NameError not defined:

from safedelete.config import *

and this line here http://django-safedelete.readthedocs.io/en/latest/admin.html needs a comma, otherwise TypeError cannot convert str to tuple object implicitly:
list_filter = ("last_name",) + SafeDeleteAdmin.list_filter

new to django, im trying to add safedelete functionality to my project and service models for my app, it is challenging, a tutorial would be nice.   heres what Im working on  https://dpaste.de/Gxc1 ,  having issues seeing my service objects - I dont have the queryset set up properly with your package Im sure.  I can see them in django admin though, highlight deleted isnt showing in red though, and a NAME is in the list_display instead of Service.   Got everything mostly working.

**Comment by Gagaro** [comment]
Hi, thanks for the fixes.

Your issue will be fixed in the next release of django-safedelete, wich will be as soon as #77 is merged.

Also in your app, you have:

```python
proj_delete = get_object_or_404(Project, pk=id).delete()
serv_delete = get_object_or_404(Service, pk=id).delete()
```

You shouldn't have the same id for two objects. Your view should take two parameters, one for each id (unless project and service are linked, in which case you should get the object the id is linked to and delete the other object from that one).

**Comment by kinganu** [comment]
yeah I revamped that and got my delete objects process working with a generic deleteview.  current set up:  https://dpaste.de/i7tE.  still have trouble dealing with permissions and not sure how i could use something like django-guardian admin views in conjunction with safedelete (prob not).    there is an issue in the query set I have using your package with [:20] at the end of it.  not sure why.  also, Id like to be able to delete a users 'safedeleted' objects that they delete, from within the admin, as super user.  but with the policy safedelete cant do that.  delete objects stay there.  maybe i need to try other policies.    should be a filter for not deleted, so view isnt cluttered up with deleted junk (testing :)).  finally deleted items should be in red, and safedelete adds a redundant name listview thats duplicating my models returned listview here:
return __str__  line of model

**Comment by Gagaro** [comment]
`highlight_deleted` is actually just `__str__` with a striked red markup if deleted.

It works in my test app, what version of Django/safedelete do you have?

**Comment by kinganu** [comment]
latest django 1.11, and pip installed safedelete.  I got it figured out for the most part, its pretty smooth, few small quirks that I can live with.  I just change the policy to HARD_DELETE if i need to delete testing junk.   My main qualm is, I cannot run 'python manage.py flush' with safedelete wrapping my models, to clear out testing data and start fresh.  heres the errors I get:

CommandError: Database djang0 couldn't be flushed. Possible reasons:
  * The database isn't running or isn't configured correctly.
  * At least one of the expected database tables doesn't exist.
  * The SQL was invalid.
Hint: Look at the output of 'django-admin sqlflush'. That's the SQL this command wasn't able to run.
The full error: relation "safedelete_safedeletemixin" does not exist

(venv)john1@j1vm/var/www/djang0/djang0$ python manage.py sqlflush
BEGIN;
TRUNCATE "auth_permission", "dashboard_service_service_type", "dashboard_service", "auth_group", "auth_user_user_permissions", "auth_user_groups", "dashboard_editcontent", "blog_post", "auth_group_permissions", "django_session", "dashboard_project", "dashboard_project_website_type", "dashboard_servicetype", "django_admin_log", "dashboard_websitetype", "auth_user", "django_content_type";
SELECT setval(pg_get_serial_sequence('"blog_post"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"dashboard_project"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"dashboard_websitetype"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"dashboard_editcontent"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"dashboard_service"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"dashboard_servicetype"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"django_admin_log"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"auth_permission"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"auth_group"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"auth_user"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"django_content_type"','id'), 1, false);
SELECT setval(pg_get_serial_sequence('"safedelete_safedeletemixin"','id'), 1, false);
COMMIT;

**Comment by Gagaro** [comment]
Did you run a makemigrations/migrate before?

**Comment by kinganu** [comment]
yeah probably. whenever I edit or make a model i run those.  not sure why safedelete is preventing a flush

**Comment by mordonez-me** [comment]
I have the same problem, was this fixed?

**Comment by Gagaro** [comment]
The problem with the admin or the flush?

**Comment by mordonez-me** [comment]
With the flash @Gagaro, I am getting the same error with safedelete_safedeletemixin.

**Comment by MattFisher** [comment]
I'm seeing a related error - after running migrate I'm trying to dumpdata but get the error:

    CommandError: Unable to serialize database: relation "safedelete_safedeletemixin" does not exist`
    LINE 1: ..."id", "safedelete_safedeletemixin"."deleted" FROM "safedelet...
                                                                 ^

**Comment by Gagaro** [comment]
Should be fixed in 0.4.3
